api/job/janitor_test.go

```diff
@@ -3,17 +3,43 @@ package job
 import (
 	"testing"
 	"time"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJanitor(t *testing.T) {
 	store := NewMemoryStore()
 	janitor := NewJanitor(store, time.Hour*24)
+	timeNow := time.Now()
```

Comment:
[nit] we usually just call this variable `now` in other places we use `time.Now()`